client/src/featureform/resources.py

```diff
@@ -307,8 +307,33 @@ def serialize(self) -> bytes:
         return bytes(json.dumps(config), "utf-8")
 
 
+@typechecked
+@dataclass
+class K8sAzureConfig:
+    account_name: str
+    account_key: str
+    container_name: str
+    path: str
+
+    def software(self) -> str:
+        return "k8s"
+
+    def type(self) -> str:
+        return "K8S_OFFLINE"
+
+    def serialize(self) -> bytes:
```

Comment:
This should serialize in the go format
type K8sConfig struct {
	ExecutorType   ExecutorType
	ExecutorConfig ExecutorConfig
	StoreType      BlobStoreType
	StoreConfig    BlobStoreConfig
}